x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheUtils.java

```diff
@@ -63,6 +63,10 @@ public static ByteRange computeRange(long rangeSize, long position, long size, l
         );
     }
 
+    public static ByteRange computeRange(long rangeSize, long position, long size) {
+        return ByteRange.of((position / rangeSize) * rangeSize, (((position + size - 1) / rangeSize) + 1) * rangeSize);
```

Comment:
Does it make sense to reuse `computeRange(rangeSize, position, size, Long.MAX_VALUE)` instead?

Reply:
I suppose it would be OK, but I prefer to have a dedicated method.